src/components/wishlist/WishListHeader.jsx

```diff
@@ -0,0 +1,21 @@
+import styled from 'styled-components';
+
+function WishListHeader() {
+  return (
+    <StyledWishListHeader>
+      <span>위시리스트</span>
+    </StyledWishListHeader>
+  );
+}
+
+export default WishListHeader;
+
+const StyledWishListHeader = styled.div`
+  position: absolute;
+  padding: 4rem 2.4rem;
+  span {
+    font-size: 20px;
+    line-height: 24px;
+    font-weight: 600;
+  }
+`;
```

Comment:
지금 div 안에 span이 들어있는 구조인데 그냥 바로 h1 사용하는 건 어떨까?
```suggestion
function WishListHeader() {
  return <StyledWishListHeader>위시리스트</StyledWishListHeader>;
}

export default WishListHeader;

const StyledWishListHeader = styled.h1`
  padding: 4rem 2.2rem;
  font-size: 2rem;
  line-height: 2.4rem;
  font-weight: 600;
  width: 100%;
`;
```

Reply:
담부터는 px 말고 rem 썼는지 한 번 더 확인해 주세용 ☺️